R/log_score.R

```diff
@@ -12,22 +12,35 @@
 #' assigned to the true outcome. Reporting the negative logarithm means that
 #' smaller values are better.
 #'
+#' The function requires users to provide observed values as a factor in order
```

Comment:
this is very good and clear. Likely worth giving some guidance on what impact reordering the factor has.

Reply:
Good idea, edited as such (in the newest PR): 

#' You could alternatively also provide a vector like
#' `observed = factor(c("a", "b", "b", "a"))` (with two levels, `a` and `b`),
#' which would result in exactly the same internal representation. Probabilities
#' then represent the probability that the outcome is equal to "b".
#' If you want your predictions to be probabilities that the outcome is "a", 
#' then you could of course make `observed` a factor with levels swapped, i.e. 
#' `observed = factor(c("a", "b", "b", "a"), levels = c("b", "a"))`